django_dynamic_theme/errors.py

```diff
@@ -5,3 +5,7 @@ class ThemeMissingError(Exception):
     """
     Is raised if not theme is set.
     """
+class NonHexValueError(Exception):
```

Comment:
Where do we use it?

Reply:
this was used in the deleted utils folders, use is present and visible now with new commit.